### ISSUE TITLE
Remove local?

### ISSUE THREAD

**Issue by IanaKhitsko** [Finderstein/MIPZ_Practice1#9]
https://github.com/Finderstein/MIPZ_Practice1/blob/7e818aa3c8111ac5eee78b833ec5759c33abaaa3/calculator.js#L140

You use haveCoinsOfThisCountry only once here, remove it. Just if (city[countryName] > 0) return true;

**Comment by Finderstein** [comment]
Our mistake. We will correct this. Thanks for your pointer

**Comment by Finderstein** [comment]
Due to another issue that is connected to this part of the code, it was rewrote. You can view changes here [cba404e](https://github.com/Finderstein/MIPZ_Practice1/commit/cba404ea0856e937dffe4cba3426493e161b0a37#diff-074b96f16fd84e394409f422b1eeb42a61fea6263645e08c8888e79990785d95R162)
